llama_stack/distribution/docker/ollama/compose.yaml

```diff
@@ -0,0 +1,46 @@
+services:
+  ollama:
+    image: ollama/ollama:latest
+    network_mode: "host"
+    volumes:
+      - ollama:/root/.ollama # this solution synchronizes with the docker volume and loads the model rocket fast
+    ports:
+      - "11434:11434"
+    devices:
```

Comment:
This might add a bit of markup to the `compose.yaml` but it might be worth creating two [profiles](https://docs.docker.com/compose/how-tos/profiles/), one that's gpu:

```
    profiles: ["gpu"]
    devices:
      - nvidia.com/gpu=all
```

and one that's CPU-only as a fallback

Reply:
Thanks for the suggestion, I'm using `podman` that do not have the `--profile` support. So workaround is to have 2 separate compose files (one for gpu, one for cpu).

Reply:
I'll create a separate distribution with separate `compose.yaml` with using ollama on CPU only. Reason is to keep it consistent and separate for distributions needing GPU & distributions on CPU (and I'm on `podman 1.0.6` without `--profile` support).